cli/pkg/exec/exec.go

```diff
@@ -0,0 +1,55 @@
+package exec
+
+import (
+	"os"
+	"os/exec"
+)
+
+func ExecGit(dir string, verbose bool) func(...string) error {
+	return func(cmds ...string) error {
+		cmd := exec.Command("git", cmds...)
+		cmd.Dir = dir
+
+		if verbose {
+			cmd.Stdout = os.Stdout
+			cmd.Stderr = os.Stderr
+		}
+
+		return cmd.Run()
+	}
+}
+
+func SetupNode(dir string, verbose bool) error {
+	// Check for nvm
+	_, ok := os.LookupEnv("NVM_DIR")
+	if ok {
+		exc(verbose, dir, "nvm", "use")
+	}
+
+	return nil
+}
```

Comment:
This function gave me some problems when running the `prepare` command initially. When running the command, I kept failing at this step with errors about the Node version: `npm ERR! code EBADENGINE`. 

We discussed this already, but we may need to expand how we handle Node environments, especially for anyone who isn't using `nvm`. Added an issue for this here: https://github.com/wordpress-mobile/release-toolkit-gutenberg-mobile/issues/157.

Reply:
Also noting that I was only able to get around an exit here by setting my Node global default, e.g., `nvm alias default v16.20.2`. It's totally fine for now to keep things moving, but we could aim to set the Node version within this command, independent of the user's global default version.

Reply:
Totally agree that this should be more robust.
 I was assuming folks would have the required node version installed to work on the project and that `nvm use` would be able to pick that up. 
On the other hand I might have had to tweak my machine to get this working back in the day.
Thanks for adding the issue!